bin/i18n/sync-up.rb

```diff
@@ -31,34 +22,15 @@ def self.parse_options
     # @option opts [true, false] :testing Whether to run in testing mode
     # @return [void]
     def self.perform(opts = parse_options)
-      crowdin_projects = opts[:testing] ? CROWDIN_TEST_PROJECTS : CROWDIN_PROJECTS
-
       I18nScriptUtils.with_synchronous_stdout do
-        puts "Sync up starting"
+        puts 'Sync up starting'
 
         I18n::Resources::Apps.sync_up(**opts)
         I18n::Resources::Dashboard.sync_up(**opts)
-
-        crowdin_projects.each do |name, options|
-          next if EXCLUDED_CROWDIN_PROJECTS.include?(name)
-
-          puts "Uploading source strings to #{name} project"
-          command = "crowdin upload sources --config #{options[:config_file]} --identity #{options[:identity_file]}"
-          Open3.popen2(command) do |_stdin, stdout, status_thread|
-            while line = stdout.gets
-              # skip lines detailing individual file upload, unless that file
-              # resulted in an unexpected response
-              next if line.start_with?("✔️  File")
```

Comment:
We are switching from Crowdin-cli to Crowdin API, I assume the outputs will look similar to the sync-down and can be managed similarly.